indexer/services/ender/src/validators/stateful-order-validator.ts

```diff
@@ -32,14 +33,20 @@ export class StatefulOrderValidator extends Validator<StatefulOrderEventV1> {
       this.event.orderRemoval === undefined &&
       this.event.conditionalOrderPlacement === undefined &&
       this.event.conditionalOrderTriggered === undefined &&
-      this.event.longTermOrderPlacement === undefined
+      this.event.longTermOrderPlacement === undefined &&
+      this.event.twapOrderPlacement === undefined
```

Comment:
We should move this into a separate function tbh, this conditional is getting long